server/openapi.yaml

```diff
@@ -0,0 +1,167 @@
+openapi: "3.0.0"
+
+info:
+  version: 0.0.3
+  title: HRXP Archive API
+
+tags:
+- name: Channels
+- name: Messages
+
+paths:
+  /channels:
+    get:
+      tags: [Channels]
+      summary: Returns a list of all channels
+      responses:
+        200:
+          description: OK; data in response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Channel'
+  /channels/{channelId}:
```

Comment:
Is this endpoint necessary? Like what will /channels/:channelId return? Just a thought. Seems like /channels/{channelId}/messages will return all the messages for a specific channel. What do you think?

Reply:
> is this endpoint necessary?

nope, I don't think there are any details about messages that we won't expose via `/channels/{channelId}/messages`. Let's remove it.

Reply:
Oops! Thought this was `/channels/{channelId}/messages/{messageId}`. I removed that in eae4f0cece580, LMK if you disagree with that.

As for `/channels/{channelId}`, I could imagine it being useful for getting details about a specific channel, but there's no reason we could just have the frontend reuse the data from `/channels`. Let's remove it as well.

Reply:
Resolved in 19d76c2.